values.yaml

```diff
@@ -794,7 +794,7 @@ client:
   # This can also be set using Helm's `--set` flag using the following syntax:
   #
   # ```shell
-  # --set 'client.extraConfig="{"log_level": "DEBUG"}"'
+  # --set 'client.extraConfig='{"log_level": "DEBUG"}"'
```

Comment:
I think this needs to be reverted. I think it was correct previously.

Reply:
I dont think the original is valid, the quotes need to be escaped.